### PR TITLE
feat: show the enrolment status in carousel cards

### PR DESCRIPTION
**Pull request by nikomakela**
HH-283

image

NOTE: It is not easy or not possible at all to colorize the output with a text-only prop, but I would consider to not even merge this, since it makes quite much noise in our frontpage. The new text does not quite fit in the card. It should probably also be a component instead of just a text so it's more manageable (in good and bad).

### CODE REVIEW

**Comment by nikomakela** [comment]
I converted this to draft, because I would like to get an approval from the designer and product owner also before the merge.

**Comment by nikomakela** [comment]
I will still update the HCRC to fix some types

- [ ] Update the HCRC

File: apps/events-helsinki/src/hooks/useRHHCConfig.tsx
Comment (nenonja) [inline apps/events-helsinki/src/hooks/useRHHCConfig.tsx:73]: what happened to `fallbackImageUrls` and `copy` settings?

Comment (nikomakela) [reply apps/events-helsinki/src/hooks/useRHHCConfig.tsx:73]: See the beginning of the config-object. They come from the `commonConfig` 
```
    return {
      ...rhhcDefaultConfig,
      ...commonConfig,
```

**Review by melniiv — APPROVED** [review]

**Comment by nikomakela** [comment]
I will now merge this so I can finally stop just updating the HCRC. It's a never ending battle - it expires so quickly.

**Pull request merged**
